### DISCUSSION TITLE
Can I see who is using which application?

### DISCUSSION THREAD

**Discussion by Tealk** [goauthentik/authentik#6077] [Q&A]
Hello,

I would be interested to know which users are using which application, is this currently possible?

**Comment by BeryJu** [comment]
You can navigate to *Events* -> *Logs* to see `Application authorized` events which store which user accessed which application

**Reply by Tealk** [reply]
But there every login is stored? 

I meant that as an overview:
Nextcloud 3 User
Zammad 5 User

**Reply by BeryJu** [reply]
authentik doesn't know which user is currently using which application, it could make an estimate based on the last time a user authorized an application and the session duration, but that data would be quite far from correct

**Reply by Tealk** [reply]
I am not interested in live data, but only in seeing whether a service is being used or not.
if not, I could stop it.

**Reply by BeryJu** [reply]
You can see historical data by going to *Applications* in the admin interface, where you'll see a graph of the logins per 8 hours over the last week which should help with this
